src/order_cmd.cpp

```diff
@@ -824,6 +922,76 @@ void OrderList::MoveOrder(int from, int to)
 	this->ReindexOrderList();
 }
 
+std::string OrderList::ToJSONString()
+{
+
+	nlohmann::json json;
+	auto& SD_data = this->GetScheduledDispatchScheduleSet();
+
+	auto& headJson = json["head"];
+	for (unsigned int i = 0; auto &SD : SD_data) {
+
+		headJson["scheduled-dispatch"][i++] = nlohmann::json::parse(SD.ToJSONString());
```

Comment:
Serialising a JSON string and then immediately parsing it suggests that a string is not really the right return type?

Reply:
I was trying to avoid having to include the json library in the headers, but I guess you are right.
Should it be changed to just use a json object then?

Reply:
That would likely be better, I agree that including the JSON library everywhere is not ideal, but I can resolve that later on